### PR TITLE
chore: Add tests to business hours and remove logs

### PR DESCRIPTION
**Pull request by KevLehman**
## Further comments

Tests added:
- https://rocketchat.atlassian.net/browse/OC-1076
- https://rocketchat.atlassian.net/browse/OC-1077
- https://rocketchat.atlassian.net/browse/OC-1082
- https://rocketchat.atlassian.net/browse/OC-1083

### CODE REVIEW

File: apps/meteor/tests/data/livechat/agent.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { api, request, credentials } from '../api-data';
+
+export const changeAgentStatus = async (agentId: string, status: string) => {
```
Comment (murtaza98) [inline apps/meteor/tests/data/livechat/agent.ts:3]: I believe we already have a helper for this API -> `makeAgentAvailable`

File: yarn.lock [hunk 2]
```diff
@@ -9784,12 +9784,6 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@rocket.chat/fuselage@link:../../../fuselage/packages/fuselage::locator=%40rocket.chat%2Fmeteor%40workspace%3Aapps%2Fmeteor":
```
Comment (murtaza98) [inline yarn.lock:9787]: Have you accidentally committed the lock file?

File: apps/meteor/tests/data/livechat/business-hours.ts [hunk 3]
```diff
@@ -2,10 +2,11 @@ import type { ILivechatBusinessHour } from '@rocket.chat/core-typings';
 import { credentials, methodCall, request } from '../api-data';
 
 export const saveBusinessHour = async (businessHour: ILivechatBusinessHour) => {
-	await request
+	const { body } = await request
```
Comment (murtaza98) [inline apps/meteor/tests/data/livechat/business-hours.ts:5]: Is this working? afaik: this method doesn't return anything

File: apps/meteor/tests/end-to-end/api/livechat/19-business-hours.ts [hunk 4]
```diff
@@ -72,7 +153,7 @@ describe('[CE] LIVECHAT - business hours', function () {
 		});
 		it('should return a just created custom business hour', async () => {
 			const name = `business-hour-${Date.now()}`;
-			await updateSetting('Livechat_business_hour_type', 'multiple');
+			await updateSetting('Livechat_business_hour_type', 'Multiple');
```
Comment (murtaza98) [inline apps/meteor/tests/end-to-end/api/livechat/19-business-hours.ts:156]:
```suggestion
			await updateSetting('Livechat_business_hour_type', LivechatBusinessHourBehaviors.MULTIPLE);
```

File: apps/meteor/tests/end-to-end/api/livechat/19-business-hours.ts [hunk 5]
```diff
@@ -43,6 +48,38 @@ describe('[CE] LIVECHAT - business hours', function () {
...
+					{
+						day: 'Monday',
+						open: true,
+						start: '00:00',
+						finish: '00:01',
+					},
+				],
+			});
+
+			// @ts-expect-error - afsdf
```
Comment (murtaza98) [inline apps/meteor/tests/end-to-end/api/livechat/19-business-hours.ts:67]: Why don't we type the response from `makeAgentAvailable` so we don't need to override the error here?

**Review by sampaiodiego — APPROVED** [review]

**Review by murtaza98 — APPROVED** [review]

**Pull request merged**